src/test/auxiliary/privacy_reexport.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-pub extern crate core;
```

Comment:
This also looks like it's also specifically testing the semantics of `pub extern crate`